packages/filesystem/src/common/filesystem.ts

```diff
@@ -17,10 +17,11 @@ export interface FileSystem extends JsonRpcServer<FileSystemClient> {
      * Returns the filestat for the given uri.
      *
      * If the uri points to a folder it will contain one level of unresolved children.
+     * The depth define the number of level for unresolved children.
      *
      * Reject if a file for the given uri does not exist.
      */
-    getFileStat(uri: string): Promise<FileStat>;
+    getFileStat(uri: string, depth: number): Promise<FileStat>;
```

Comment:
Without assuming anything about the implementation, what should happen if I pass in `2` or `3`?

Reply:
Can we have an optional (or default) value maybe?